### PR TITLE
Cookie cutter boilerplate.

### PR DESCRIPTION
**Pull request by davidjoy**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "presets": [
+    [
+      "env",
+      {
+        "targets": {
+          "browsers": ["last 2 versions", "ie 11"]
```
Comment (albemarle) [inline .babelrc]: 😿

File: docs/decisions/0001-record-architecture-decisions.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
+1. Record Architecture Decisions
```
Comment (albemarle) [inline docs/decisions/0001-record-architecture-decisions.rst]: Are we planning on having this in every repo?

Comment (davidjoy) [reply docs/decisions/0001-record-architecture-decisions.rst]: I considered it a subtle reminder to remember to write them... but don't feel strongly.

Comment (davidjoy) [reply docs/decisions/0001-record-architecture-decisions.rst]: (I had to go hunt the directory structure and 0001 ADR down when I wanted to add one to profile)

**Pull request merged**